weather_mv/loader_pipeline/ee.py

```diff
@@ -261,6 +261,8 @@ def validate_arguments(cls, known_args: argparse.Namespace, pipeline_args: t.Lis
         pipeline_options = PipelineOptions(pipeline_args)
         pipeline_options_dict = pipeline_options.get_all_options()
 
+        assert not known_args.zarr, 'Reading Zarr is not (yet) supported.'
```

Comment:
It seems weird to assert here where you raise more specific errors in other files...